readme.md

```diff
@@ -20,38 +20,44 @@ __Magnolia__ is a generic macro for automatic materialization of typeclasses for
 Given an ADT such as,
 ```scala
 enum Tree[+T]:
```

Comment:
Wouldn't adding `derives` clause to the enum be more 'user friendly' (than an explicit given)?
```suggestion
enum Tree[+T] derives Print:
  case Branch(left: Tree[T], right: Tree[T])
  case Leaf(value: T)
```

Reply:
Definitely! :) I have changed it according to Your suggestion :)